infrastructure/variables.tf

```diff
@@ -49,6 +49,10 @@ variable "business_area" {
   default = "cft"
 }
 
-variable jenkins_AAD_objectId {}
+variable "pgsql_storage_mb" {
+  default = 65536
+}
```

Comment:
This could change it everywhere?  Were we advised to do that?

Reply:
This is the default size https://github.com/hmcts/terraform-module-postgresql-flexible?tab=readme-ov-file#input_pgsql_storage_mb so I'm defining it here and then setting the size it got upgraded to in Azure in demo.tfvars

Reply:
The default should be set already in the terraform module so we shouldn't need to set it ourselves (you can prob find it in the module repo).  You should therefore just be able to override in demo where we want to change it?

Hopefully that will avoid all the destroy labels in prod etc.

Reply:
When I tried to remove the default form the file I got a this error, "Error: Reference to undeclared input variable", looks like I'll have to have it set.